JavaScript/CodingConventions.md

```diff
@@ -532,6 +581,16 @@ var MY_CONST = 2;
 var myConst = 2;
 ```
 
+Constants exists in ECMAScript6+, so you can use `const` for declaration. Still keep them camelCase:
```

Comment:
:oncoming_police_car: **Wikia Day Police notice:** Please turn off Jira, GitHub and even your Gmail if it distracts you from editing wikias! :japanese_ogre: